Install test fails on CygWin with py36; pywbem_os_setup scripts not in source dist archive

pywbem 0.17 still uses M2Crypto which does not ship as a wheel so it needs openssl header files at install time. Pywbem deals with this by providing the pywbem_os_setup.sh/.bat scripts.

There are two issues related to that:
* The recently introduced install test missed that and does not invoke these scripts. This leads to an install failure on CygWin with Python 3.6 and minimum package levels (I'm actually surprised that happened only in one of our environments). See this Appveyor build: https://ci.appveyor.com/project/andy-maier/pywbem/builds/32152461. Failing compilation step during install of M2Crypto:
```
  gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -ggdb -O2 -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -fdebug-prefix-map=/usr/src/ports/python36/python36-3.6.9-1.x86_64/build=/usr/src/debug/python36-3.6.9-1 -fdebug-prefix-map=/usr/src/ports/python36/python36-3.6.9-1.x86_64/src/Python-3.6.9=/usr/src/debug/python36-3.6.9-1 -ggdb -O2 -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -fdebug-prefix-map=/usr/src/ports/python36/python36-3.6.9-1.x86_64/build=/usr/src/debug/python36-3.6.9-1 -fdebug-prefix-map=/usr/src/ports/python36/python36-3.6.9-1.x86_64/src/Python-3.6.9=/usr/src/debug/python36-3.6.9-1 -I/usr/include/python3.6m -I/tmp/pip-build-n03pn9po/M2Crypto/SWIG -c SWIG/_m2crypto_wrap.c -o build/temp.cygwin-3.1.4-x86_64-3.6/SWIG/_m2crypto_wrap.o -DTHREADING -Wno-deprecated-declarations
  SWIG/_m2crypto_wrap.c:3557:10: fatal error: openssl/err.h: No such file or directory
   3557 | #include <openssl/err.h>
        |          ^~~~~~~~~~~~~~~
  compilation terminated.
  error: command 'gcc' failed with exit status 1
```

* The source distribution archive does not contain these scripts, but probably should.

No roll forward into master needed, because there we have dropped M2Crypto.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -4,6 +4,8 @@ include README.rst
 include README_PYPI.rst
 include INSTALL.md
 include requirements.txt
+include pywbem_os_setup.sh
+include pywbem_os_setup.bat
 include setup.py
 include build_moftab.py
 include mof_compiler

--- a/docs/changes.rst
+++ b/docs/changes.rst
@@ -32,6 +32,12 @@ Released: not yet
   versions that support a particular Python versions beyond that.
   This only affects development of pywbem. (See issue #2174)
 
+* Setup: Added the scripts for installing OS-level dependencies
+  (pywbem_os_setup.sh/.bat) to the source distribution archive. Note that
+  starting with the upcoming pywbem 1.0.0, these scripts are no longer needed,
+  so this change will not be rolled forward into 1.0.0.
+  (See issue #2173)
+
 **Enhancements:**
 
 **Cleanup:**

--- a/makefile
+++ b/makefile
@@ -287,6 +287,8 @@ dist_manifest_in_files := \
     README_PYPI.rst \
     INSTALL.md \
     requirements.txt \
+    pywbem_os_setup.sh \
+    pywbem_os_setup.bat \
     setup.py \
     build_moftab.py \
     mof_compiler \
@@ -305,6 +307,8 @@ dist_dependent_files := \
     README_PYPI.rst \
     INSTALL.md \
     requirements.txt \
+    pywbem_os_setup.sh \
+    pywbem_os_setup.bat \
     setup.py \
     build_moftab.py \
     mof_compiler \

